weed/server/volume_grpc_copy.go

```diff
@@ -108,7 +108,8 @@ func (vs *VolumeServer) VolumeCopy(req *volume_server_pb.VolumeCopyRequest, stre
 		nextReportTarget := reportInterval
 		var modifiedTsNs int64
 		var sendErr error
-		if modifiedTsNs, err = vs.doCopyFile(client, false, req.Collection, req.VolumeId, volFileInfoResp.CompactionRevision, volFileInfoResp.DatFileSize, dataBaseFileName, ".dat", false, true, func(processed int64) bool {
+		throttler := util.NewWriteThrottler(req.IoBytePerSecond)
```

Comment:
use the default value if zero

Reply:
Do you mean to add a parameter to VolumeServer and give a default value?

Reply:
vs.compactionBytePerSecond should take priority over req.IoBytePerSecond